src/fiatconnect/ReviewScreen.test.tsx

```diff
@@ -137,25 +136,6 @@ describe('ReviewScreen', () => {
       expect(queryByTestId('expiredQuoteDialog')?.props.visible).toEqual(true)
     })
 
-    it('shows expired dialog when submitting expired quote', async () => {
-      const expirySecs = 1
-      const mockProps = getProps(CICOFlow.CashOut, false, CryptoType.cUSD, expirySecs)
-      const store = createMockStore()
-      const { getByTestId, queryByTestId } = render(
-        <Provider store={store}>
-          <FiatConnectReviewScreen {...mockProps} />
-        </Provider>
-      )
-
-      expect(queryByTestId('expiredQuoteDialog')?.props.visible).toEqual(false)
-
-      await setTimeout(expirySecs * 1000)
```

Comment:
I couldn't use setTimeout here as it caused one of the CI tests to fail. I tried sleeping by awaiting a promise wrapped around setTimeout, but it seemed to never timeout oddly enough. I also tried messing with jest.setSystemTime but that made both tests fail. Please let me know if you have any insights as to what's going on.

Reply:
Nevermind, figured it out!